hack/deploy_vault_config.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
```

Comment:
Why we need this? IMO I believe that it is better to maintain the config file with the config_updater plugin and mount it to the vault as configmap.

Reply:
Because vault does not support to read this from a file